t/15-shared-plugin-gru.t

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env perl
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+use Test::Most;
+
+use Test::Warnings ':report_warnings';
+use Test::MockModule;
+use Test::MockObject;
+use Test::Mojo;
+use FindBin;
+use lib "$FindBin::Bin/lib", "$FindBin::Bin/../external/os-autoinst-common/lib";
+use OpenQA::Test::TimeLimit '5';
+use OpenQA::Test::Case;
+use OpenQA::Shared::Plugin::Gru;    # SUT
+
+ok my $gru = OpenQA::Shared::Plugin::Gru->new, 'can instantiate gru plugin';
+
+my %connect_info = (dsn => 'my_dsn', user => 'user', password => 'foo');
+my $storage = Test::MockObject->new->set_always(connect_info => [\%connect_info]);
+my $schema = Test::MockObject->new->set_always(storage => $storage)->set_true('search_path_for_tests');
+my %config = (misc_limits => {minion_job_max_age => 0});
+my $minion = Test::MockObject->new->set_true('on');
+my $log = Test::MockObject->new->set_true('level', 'info');
```

Comment:
Just instantiating a real object via `Mojo::Log->new` would probably not hurt (but considering we only need two methods here mocking it is ok I suppose).

Reply:
Yes, and it's the most efficient for this unit-level test.